PurchasesTests/Purchasing/__Snapshots__/ProductInfoTests/testEncoding.1.json

```diff
@@ -6,21 +6,21 @@
     {
       "offer_identifier" : "offerid1",
       "payment_mode" : 0,
-      "price" : 11
+      "price" : 11.199999999999999
     },
     {
       "offer_identifier" : "offerid2",
       "payment_mode" : 1,
-      "price" : 12
+      "price" : 12.199999999999999
```

Comment:
Confirmed: this is just the way `Decimal: Encodable` is implemented (funny that `NSDecimalNumber` avoids this problem). It's just ugly to humans but irrelevant to computers so 🤷‍♂️